triggers/simulations/contention.rst

```diff
@@ -0,0 +1,248 @@
+.. Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Simulation Use Case Example: Impact of Uncontrolled Contention
+==============================================================
+
+As triggers interact with the ledger asynchronously and the ledger *may* also process transactions concurrently, there is a high risk that multiple changes (e.g. exercises, archives, etc.) to the same contract might overlap each other. When this happens, contention is said to occur.
```

Comment:
Suggestion. We could say this is the case for any legder client, not only triggers ?